src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub mod hellogo_fares;
 #[cfg(feature = "proj")]
 pub mod kv1;
 pub mod merge_stop_areas;
-mod minidom_utils;
+pub(crate) mod minidom_utils;
```

Comment:
This `minidom_utils` can be exposed public for our crate because it might be used anywhere in our crate. However, we don't want to expose it to the world as an API of `transit_model`. `pub(crate)` just do that.
https://doc.rust-lang.org/stable/reference/visibility-and-privacy.html#pubin-path-pubcrate-pubsuper-and-pubself

Reply:
I just wonder if all the `pub(crate)` are very useful here. 
The code should compile without and the modules are not accessible from outside because they are private

Reply:
It's just a way to document the fact that this module is a utility for the entire crate. Note that we already do use `pub(crate)` for `common_format` and `utils`. But there is inconsistencies with `add_prefix`, `read_utils` and `netex_utils` which are private. Do you want me to move all of them to the same state? If yes, which one do you prefer: `mod` or `pub(crate) mod`?

Reply:
> Do you want me to move all of them to the same state?
Yes if we are sure that `mod` and `pub(crate) mod` do the same in this case.

> which one do you prefer
I prefer `mod` but it's a personal choice.
We can ask @ArnaudOggy and @patochectp for advice

Reply:
I prefer ```mod``` too, but without more conviction or opinion.

Reply:
Done in a75ea2a6d7a9d48b64d6fa8f8a788d344e10eede.